stdlib/test/collections/test_dict.mojo

```diff
@@ -182,14 +182,14 @@ def test_dict_copy_add_new_item():
 
 def test_dict_copy_calls_copy_constructor():
     var orig = Dict[String, CopyCounter]()
-    orig["a"] = CopyCounter()^
+    orig["a"] = CopyCounter() ^
 
     # test values copied to new Dict
     var copy = Dict(orig)
     # I _may_ have thoughts about where our performance issues
     # are coming from :)
-    assert_equal(5, orig["a"].copy_count)
-    assert_equal(6, copy["a"].copy_count)
+    assert_equal(4, orig["a"].copy_count)
```

Comment:
The copy count went down by 1 from these changes. Is that expected?

Reply:
Yep, this is expected since we no longer return a copy of the `T` from `Optional`.  Slowly getting our copies down to a more sane amount in `Dict` which is one of the perf problems in `Dict` right now. 😄